test/e2e/framework/util.go

```diff
@@ -2237,19 +2256,26 @@ func (f *Framework) testContainerOutputMatcher(scenarioName string,
 		err = nil
 		logs, err = GetPodLogs(f.Client, ns, pod.Name, containerName)
 		if err != nil {
-			By(fmt.Sprintf("Warning: Failed to get logs from node %q pod %q container %q. %v",
-				podStatus.Spec.NodeName, podStatus.Name, containerName, err))
+			Logf("Warning: Failed to get logs from node %q pod %q container %q. %v",
+				podStatus.Spec.NodeName, podStatus.Name, containerName, err)
 			time.Sleep(5 * time.Second)
-			continue
-
+		} else {
+			Logf("Successfully fetched pod logs:%v\n", logs)
+			break
```

Comment:
Actually, do we really need this retry loop? We've already waited for the containers to succeed.